tests/Type/TestClasses/OtherEnumType.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GraphQL\Tests\Type\TestClasses;
+
+use GraphQL\Language\AST\Node;
+use GraphQL\Type\Definition\EnumType;
+
+class OtherEnumType extends EnumType
+{
+    public function __construct()
+    {
+        $config = [
+            'name'   => 'OtherEnum',
+            'values' => [
+                'ONE',
+                'TWO',
+                'THREE',
+            ],
+        ];
+        parent::__construct($config);
+    }
+
+    /**
+     * @param mixed[]|null $variables
```

Comment:
```suggestion
     * @param array<string, mixed>|null $variables
```

Reply:
Not possible because of 
```
  28     Parameter #2 $variables (array<string, mixed>|null) of method GraphQL\Tests\Type\TestClasses\OtherEnumType::parseLiteral() should be contravariant with parameter
         $variables (array|null) of method GraphQL\Type\Definition\EnumType::parseLiteral()
  28     Parameter #2 $variables (array<string, mixed>|null) of method GraphQL\Tests\Type\TestClasses\OtherEnumType::parseLiteral() should be contravariant with parameter
         $variables (array|null) of method GraphQL\Type\Definition\LeafType::parseLiteral()
```